### DISCUSSION TITLE
Welcome to Plaid2QIF Discussions!

### DISCUSSION THREAD

**Discussion by ebridges** [ebridges/plaid2qif#18] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this product.

Thanks!

**Comment by ZenDevMaster** [comment]
Appreciate the development work that went into this project, thanks! Would be cool if it could somehow be wrapped into an automated import task
